mydocstring/parse.py

```diff
@@ -42,9 +42,9 @@ class DocString(object):
 
     Attributes:
         delimiter: A string that is used to identify a section and is placed
-            after the section name (e.g., `Arguments:`). Defaults to `': '`.
+            after the section name (e.g., `Arguments:`). Defaults to `':'`.
         arg_delimiter: A string that specifies how to separate arguments in a
-            argument list. Defaults to `':'`.
+            argument list. Defaults to `': '`.
```

Comment:
why remove a space on line 45 and add one on line 47? let's be consistent and do the same on both lines, no space.

Reply:
I had to interchange the spaces due to these lines of code:
https://github.com/ooreilly/mydocstring/blob/0eaf978effa77e52ec3e8b6b17b235e00fb84016/mydocstring/parse.py#L91-#L92

Reply:
I see. Thanks for making the change so that the documentation is consistent with the code.